test/deprecated.jl

```diff
@@ -1,8 +1,9 @@
 using DataFrames, FileIO, RDatasets, Compat
-fn = tempname()
-open(fn, "w") do f
-    redirect_stderr(f) do
-        dataset("datasets", "iris")
+mktemp() do fn
```

Comment:
Didn't know about `mktemp`. Anyway, I had to close the file before I could call `readstring` so `mktemp` might not work in this case.